RocketChat/Rocket.Chat.ReactNative
# [IMPROVE] Fetch members from API endpoint

## Proposed changes

Currently we've been using a method call for fetching members, on `RoomMembersView`, but we should use REST API like the Web client does.

## Issue(s)	


## How to test or reproduce


In order to test this you should create a group, direct message with multiple users or team, and server version being at least 3.16.0

1. Go to a room
2. Tap on room's title
3. Tap on Members

It should render the members that are online from that room

## Screenshots

## Types of changes




## Checklist



## Further comments
